### ISSUE TITLE
Desktop version of vial-gui cannot recognize tinygo-keyboard

### ISSUE THREAD

**Issue by sago35** [sago35/tinygo-keyboard#20]
The web version, vial-web (https://vial.rocks/), recognizes tinygo-keyboard, but vial-gui does not.

**Comment by sago35** [comment]
To be recognized by vial-gui, it seems necessary to include the Vial serial magic (vial:f64c2b3c) in the usb.Serial (STRING DESCRIPTOR).
However, that feature is not available in TinyGo 0.30.

https://github.com/tinygo-org/tinygo/blob/v0.30.0/src/machine/usb.go#L162-L164

```go
		case usb.ISERIAL:
			// TODO: allow returning a product serial number
			SendZlp()
```

**Comment by sago35** [comment]
I have created the following PR, and if it gets merged, it will be usable with TinyGo 0.31.

https://github.com/tinygo-org/tinygo/pull/3989

**Comment by sago35** [comment]
The following PR has been merged. With the latest dev version, you can rewrite ISERIAL, and it will be recognized by vial-gui.

https://github.com/tinygo-org/tinygo/pull/3989
